docs/navigate.md

```diff
@@ -23,9 +24,12 @@ The plugin supports these options:
         frameRate: 60,
         mode: "smart",       // enable smart pan mode
         enableTouch: false,
-        touchMode: ""
+        touchMode: "",
+        allowRecenter: true
```

Comment:
I'm backtracking on this suggestion (see below).

Reply:
Done.